django_orghierarchy/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+
+from .models import Organization
+
+
+class OrganizationForm(forms.ModelForm):
+
+    class Meta:
+        model = Organization
+        fields = (
+            'data_source', 'origin_id', 'classification',
+            'name', 'founding_date', 'dissolution_date',
+            'parent', 'affiliated',
```

Comment:
As here we have the field for `parent` of a suborganization, similarly we should actually not have the `affiliated` field, but instead the `responsible_organization` field that tells which organization this one is affiliated to. :)